daemon/delete_windows.go

```diff
@@ -0,0 +1,24 @@
+// +build windows
+
+package daemon
+
+import (
+	"fmt"
+)
+
+// VolumeRm removes the volume with the given name.
+// If the volume is referenced by a container it is not removed
+// This is called directly from the remote API
+func (daemon *Daemon) VolumeRm(name string) error {
+	v, err := daemon.volumes.Get(name)
```

Comment:
This method should be empty for windows and just `return nil` as @jhowardmsft comments. Volumes don't work on windows.